gallery/LICENSE

```diff
@@ -0,0 +1,324 @@
+// Copyright 2019 The Flutter team. All rights reserved.
```

Comment:
We've got a license file for the repo, so this can be left out.

Reply:
Removed